lib/membrane_ogg/payloader/opus.ex

```diff
@@ -25,6 +25,15 @@ defmodule Membrane.Ogg.Payloader.Opus do
                 See https://datatracker.ietf.org/doc/html/rfc7845#section-4
                 """
               ],
+              random_serial_number?: [
+                type: :non_neg_integer,
+                default: true,
+                description: """
+                Ogg logical bitstreams must be assigned a unique 4-byte serial number which is chosen randomly.
+                This option allows you to enforce using the same serial number every time which can be useful for reproducability.
+                See https://datatracker.ietf.org/doc/html/rfc3533#section-4
+                """
```

Comment:
`type` is some sort of helper for the parser, not the actual typespec of the function. It will insert the default one, but it's better to just use the spec.

I would consider also consider the following modification. I think it would make sense to allow passing the serial number as the option, instead of hardcoding it
```suggestion
              serial_number: [
                spec: non_negative_integer() | :random,
                default: :random,
                description: """
                Ogg logical bitstreams must be assigned a unique 4-byte serial number which is chosen randomly.
                This option allows you to enforce using the same serial number every time which can be useful for reproducability.
                See https://datatracker.ietf.org/doc/html/rfc3533#section-4
                """
```